core/server/worker/src/main/java/alluxio/worker/block/BlockMetadataManager.java

```diff
@@ -81,19 +79,13 @@ private BlockMetadataManager() {
         mTiers.add(tier);
         mAliasToTiers.put(tier.getTierAlias(), tier);
       }
-      // Create the block iterator.
+
       if (ServerConfiguration.isSet(PropertyKey.WORKER_EVICTOR_CLASS)) {
         LOG.warn(String.format("Evictor is being emulated. Please use %s instead.",
             PropertyKey.Name.WORKER_BLOCK_ANNOTATOR_CLASS));
-        // Create emulating block iterator.
-        BlockMetadataEvictorView initManagerView = new BlockMetadataEvictorView(this,
-            Collections.<Long>emptySet(), Collections.<Long>emptySet());
-        mBlockIterator = new EmulatingBlockIterator(this,
```

Comment:
Emulation needs to be kept for user defined evictors until next major version.

When a deprecated evictor is set, I recommend mapping old evictor classes to corresponding block-annotator classes here. 

For example; if evictor class is set to `alluxio.worker.block.evictor.LRFUEvictor`, our compensation is ideally to set block-annotator class as `alluxio.worker.block.annotator.LRFUAnnotator`, along with other accompanying configurations like `*_LRFU_ATTENUATION_FACTOR` and `*_LRFU_STEP_FACTOR`.

Reply:
For some evictor like alluxio.worker.block.evictor.GreedyEvictor, there is no corresponding annotator. It's hard to map to existed annotator. What's your opinion about it?

Reply:
Emulation would be the only choice for those and 3rd party evictors.

Reply:
Just pushed a new commit, the current mapping rule is:
 - LRFUEvictor --> LRFUAnnotator. (With default LRFU_ATTENUATION_FACTOR and LRFU_STEP_FACTOR)
 - LRUEvictor /PartialLRUEvictor/GreedyEvictor --> LRUAnnotator; (The GreedyEvictor is more like a demo evictor, seems no need to create a new GreedyAnnotator and corresponds it with GreedyEvictor, so maybe we can use LRU Annotator instead)
 - User Defined Evictor --> EmulatingBlockIterator

Reply:
Hi @ggezer, please let me know if further modification is needed, thanks